### DISCUSSION TITLE
关于顶栏

### DISCUSSION THREAD

**Discussion by Cauliflower12315** [the1812/Bilibili-Evolved#2910] [提问 / Q&A]
①v1和v2现在都会有在顶栏出现这种情况
![image](https://user-images.githubusercontent.com/96857365/150345484-fa7b7032-4879-406d-baf4-74be233060b6.png)
而且这在首页显示是正常的，进入一个视频才出现了这种情况。且我创了一个小号发现并没有类似的情况，令人疑惑。
②v2搜索框确实存在首页和其他地方（视频内、他人主页内等）显示不同的情况
![image](https://user-images.githubusercontent.com/96857365/150346168-d8c68795-82a0-402f-af2b-f378bb9ceb65.png)
![image](https://user-images.githubusercontent.com/96857365/150346441-012db4ec-5ebf-46df-99af-ab4ada623f85.png)
![image](https://user-images.githubusercontent.com/96857365/150346515-99e379f7-35a5-4758-be99-511a205a1137.png)
谢谢谢谢

**Comment by Geeyun-JY3** [comment]
1. 请先善用搜索：#2740、#2893

**Reply by Cauliflower12315** [reply]
好的谢谢
